File: Java/Laba8/src/main/java/ua/lviv/iot/goods/goods/Book.java
package ua.lviv.iot.goods.goods;

import ua.lviv.iot.goods.enums.CategoryEnum;

public class Book extends Good {
    private String author;


    public Book(final Integer id, String name, CategoryEnum categoryEnum, boolean aviability, int price, String author, int pages) {
        super(id, name, categoryEnum, aviability, price, author, pages);
        this.author = author;
    }

    public String getAuthor() {
        return author;
    }

    public void setAuthor(String author) {
        this.author = author;
    }


    @Override
    public String getHeaders() {
        return super.getHeaders() + "author";
    }

    @Override
    public String toCSV() {
        return super.toCSV() + author;
    }

}

File: Python/lab11/goods/Book.py
from goods.Good import Good


class Book(Good):
    def __init__(self, name, category, price, author, pages):
        super().__init__(name, category, price, author, pages)

    def __str__(self):
        return ", Name:" + str(self.name) + " Category: " + str(self.category) + ", Price:" + str(
            self.price) + ", Author:" + str(self.author) + ", Pages:" + str(self.pages)

